clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/TitanConfiguration.groovy

```diff
@@ -45,6 +46,12 @@ class TitanConfiguration {
   }
 
   @Bean
+  @DependsOn('netflixTitanCredentials')
+  TitanDeployHandler titanDeployHandler(TitanClientProvider titanClientProvider) {
```

Comment:
Yeah I have some additional Titus work coming so I just wanted to limit this change set to what played nice with sed edits